frontend/server/src/Controllers/Session.php

```diff
@@ -677,6 +810,54 @@ private static function ThirdPartyLogin(
         return ['status' => 'ok'];
     }
 
+    private static function thirdPartyLogin(
```

Comment:
```suggestion
    /**
     * @never-return
     */
    private static function thirdPartyLogin(
```

también se puede agregar esa anotación a las funciones que llaman esto al final incondicionalmente?